tests/_ast/test_app.py

```diff
@@ -293,3 +293,19 @@ def _(__x__: int) -> None:
         configs = tuple(app._configs())
         assert configs[0].disabled
         assert configs[1].hide_code
+
+    @staticmethod
+    def test_condition_definition() -> None:
```

Comment:
can we include a test case from the actual issue?
https://github.com/marimo-team/marimo/issues/599

Reply:
```
@app.cell
def __():
    objects =  iter([])

    # Wrap the queryset with tqdm for a progress bar
    for obj in objects:
        pass
    return (
        obj,
        objects,
    )
```